main.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+import config
+import util.dht
+import argparse
+import typing
+import RPi.GPIO as GPIO
+
+class Args(typing.NamedTuple):
+    configs_path: str
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-configs_path', default='data/configs.json')
+    args = parser.parse_args()
+    return Args(args.configs_path)
+
+
+def event_loop(args: Args):
+    configs = config.load_configs(args.configs_path)
+    loop = asyncio.get_event_loop()
```

Comment:
I'm getting a `DeprecationWarning: There is no current event loop`. By the looks of it, it seems to be a Python 3.11 thing.

I found [this](https://bobbyhadz.com/blog/deprecationwarning-there-is-no-current-event-loop#:~:text=The%20message%20%22DeprecationWarning%3A%20There%20is,when%20the%20warning%20is%20shown.) solution, which is supposed to be portable:
```python
if sys.version_info < (3, 10):
    loop = asyncio.get_event_loop()
else:
    try:
        loop = asyncio.get_running_loop()
    except RuntimeError:
        loop = asyncio.new_event_loop()

    asyncio.set_event_loop(loop)
```
but I'm concerned that the Pi's have an older python version installed, so I might just downgrade locally.